aladdin/commands/start.py

```diff
@@ -79,9 +71,7 @@ def start(
     # Start command values
     values = {
         "deploy.imageTag": "local",
-        "deploy.mountPath": pc.mount_path,
         "deploy.namespace": namespace,
-        "deploy.withMount": with_mount,
```

Comment:
I guess this shouldn't break too many helm charts (if any) since a missing value will just be treated as "falsy"